api-docs/index.rst

```diff
@@ -13,4 +13,51 @@ What does EVA do?
 
 * EVA **improves throughput** by introducing sampling, filtering, and caching techniques.
 
-* EVA **improves accuracy** by introducing state-of-the-art model specialization and selection algorithms.
\ No newline at end of file
+* EVA **improves accuracy** by introducing state-of-the-art model specialization and selection algorithms.
+
+Starting EVA
+------------
+
+.. grid:: 1 1 2 2
+    :gutter: 3
+    :margin: 0
+    :padding: 3 4 0 0
+
+    .. grid-item-card:: :doc:`Tutorial: Intro to EVA <source/tutorials/tutorials>`
+        :link: source/tutorials/tutorials
+        :link-type: doc
+        :class-card: content-cards
+
+        A simple example of using EVA.
+
+    .. grid-item-card:: :doc:`SQL Commands <source/reference/index>`
+        :link: source/reference/index
+        :link-type: doc
+        :class-card: content-cards
+
+        All SQL commands supported by EVA.
+    
+    .. grid-item-card:: :doc:`Registering UDFs <source/reference/udf>`
+        :link: source/reference/udf
+        :link-type: doc
+        :class-card: content-cards
+
+        A step-by-step tour of creating custom User Defined Functions for EVA.
+
+
+
+
+.. spelling::
+
+.. .. |pypi_status| image:: https://img.shields.io/pypi/v/bentoml.svg?style=flat-square
+..    :target: https://pypi.org/project/evadb
+.. .. |downloads| image:: https://pepy.tech/badge/bentoml?style=flat-square

```

Comment:
Do we need `bentoml` links here?

Reply:
No, they are not necessary. I have left them as they may be useful in the future for adding github links, pypi status, etc.